src/app/InteractionModelEngine.h

```diff
@@ -255,26 +255,18 @@ void DispatchSingleClusterResponseCommand(const ConcreteCommandPath & aCommandPa
 bool ServerClusterCommandExists(const ConcreteCommandPath & aCommandPath);
 
 /**
- *  Fetch attribute value and version info and write to the TLVWriter provided.
- *  When the endpoint / cluster / attribute / event data specified by aClusterInfo does not exist, corresponding interaction model
- * error code will be put into the writer, and CHIP_NO_ERROR will be returned and apDataExists will be set to false.
- *  If the data exists on the server, the data (with tag kCsTag_Data) and the data version (with tag kCsTag_DataVersion) will be put
- * into the TLVWriter and apDataExists will be set to true. TLVWriter error will be returned if any error occurred during encoding
- * these values.
+ *  Fetch attribute value and version info and write to the AttributeReport provided.
```

Comment:
The existing comments around error generation are still valid, but perhaps, need to get re-worded in the context of `AttributeReportIB`.

Reply:
agree, just reword regarding error part, thanks